app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: Car) -> int:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income = income + self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        wash_price = car.comfort_class * \
+            (self.clean_power - car.clean_mark) * \
+            (self.average_rating / self.distance_from_city_center)
```

Comment:
Ident this calculation in line with an equal sign and place calculation signs not at the end of a line but at the beginning as mentioned in the [checklist](https://github.com/CeO-PsH/py-car-wash-station/blob/developer/checklist.md)